src/ToolBox/SOS/lldbplugin/CMakeLists.txt

```diff
@@ -17,6 +17,11 @@ elseif(CLR_CMAKE_PLATFORM_ARCH_ARM64)
     return()
 endif()
 
+if(CLR_CMAKE_PLATFORM_LINUX_ALPINE)
+    message(WARNING "lldb is not supported on Alpine Linux yet")
```

Comment:
Not sure the best way forward, guess either just make it optional everywhere or have a nolldb parameter to build.sh . Just wanted to move forward. `CLR_CMAKE_PLATFORM_LINUX_ALPINE` should probably be a musl toggle.